### ISSUE TITLE
Navigation buttons not working if wizard has nested router path

### ISSUE THREAD

**Issue by kvanrobbroeck** [cmdap/ng-wizard#1]
Hello,

I have a problem getting the router navigation buttons to work when the wizard is mounted on a nested path:

```
const routes: Routes = [
  { path: 'filament', component: NgWizardComponent, children: [
      { path: 'material', component: MaterialStepComponent },
      { path: 'extruder', component: ExtruderStepComponent },
      { path: 'temperature', component: TemperatureStepComponent },
      { path: '**', redirectTo: 'material' },
    ]},
  { path: '**', component: HomeComponent },
];
```

Using the buttons seems to want to ignore `filament` so instead of going to the next step as `/filament/extruder` to goes to `/extruder`.

All other stuff works (I can go directly to a step by URL, etc...

I would like to contribute to this project, if you can point me in the right direction, I'm happy to submit a pull request.

**Comment by jbrems** [comment]
Hi @kvanrobbroeck ,

The situation you are describing should be handled by the code of the `NgWizardService.navigateToStep` method.

```
if (this.wizardRoute.path) {
  return this.router.navigate([stepData.path], { relativeTo: this.route });
}
return this.router.navigate([stepData.path]);
```

Your case should satisfy the if statement and add the `relativeTo` option to the `Router.navigate` method.

If you are willing to contribute you may figure out why `this.wizardRoute.path` is falsy or why the `relativeTo` option does not seem to work (this.route might be undefined).

Thanks in advance,
Jonas (in name of CM's DAP team)

**Comment by jbrems** [comment]
There seems to be an issue with using the Router's relativeTo option in combination with using the HashLocationStrategy. A workaround has been implemented.

This issue should be fixed in version 1.4.1.
